src/components/DeviceChecker/DeviceChecker.vue

```diff
@@ -472,4 +458,88 @@ export default {
 ::v-deep .modal-container {
 	display: flex !important;
 }
+
+.modal-header{
+	display: flex;
+}
+
+.adminly-close-button {
+	margin-left: auto;
+	background-image: url("../../../img/close.svg");
+	background-position: center;
+	background-repeat: no-repeat;
+	padding-inline: 1rem !important;
+	background-color: white;
+	box-shadow: none;
+    border: none;
+}
+
+.preview__novideo{
+	height: 100%;
+	background-image: url("../../../img/user.svg");
+	background-position: center;
+	background-repeat: no-repeat;
+}
+
+.device-checker__preview{
+	background-color: #9195EA4D !important;
+	width: 300px !important;
```

Comment:
By setting the width, the preview box is not totally centered in the modal. It would be better to not specify the width.